flink-core/src/main/java/org/apache/flink/api/common/ArchivedExecutionConfig.java

```diff
@@ -54,6 +54,19 @@ public ArchivedExecutionConfig(ExecutionConfig ec) {
 		}
 	}
 
+	public ArchivedExecutionConfig(
```

Comment:
Sorry, i was mixing things up. I thought i would have to change the constructors of other Archived* classes as well to be consistent, but they already had a constructor like this.